inspirehep/modules/workflows/views/callback.py

```diff
@@ -97,7 +148,13 @@ def webcoll_callback():
 
     Expects the request data to contain a list of record ids in the
     recids field.
+
+    DEPRECATED: replaced by the simpler continue_workflow_callback
```

Comment:
It works on a daily basis on labs.inspirehep.net (https://github.com/inspirehep/inspire-next/blob/production/inspirehep/modules/workflows/views/callback.py) - see `production` branch. Since the code on legacy has not changed, maybe something is not right in the `master` branch and needs fixing.